### PR TITLE
CMR-9774: Updating the CMR 

### PR DESCRIPTION
**Pull request by eereiter**
# Overview

### What is the feature/fix?
Changing the acl software to mitigate linter findings.

### What is the Solution?
Removing not needed required statements and putting meaningless declarations in front of every macro so the linter doesn't complain about them. .

### What areas of the application does this impact?
Acl library and all other dependencies.

# Checklist

- [x ] I have updated/added unit and integration tests that prove my fix is effective or that my feature works
- [x ] New and existing unit and int tests pass locally and remotely with my changes
- [x ] I have removed unnecessary/dead code and imports in files I have changed
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

File: umm-lib/resources/schema/iso_mends/resources/Codelist/eosCodelists.xml [hunk 1]
```diff
@@ -187,4 +187,53 @@
          </gmx:codeEntry>
       </gmx:CodeListDictionary>
    </gmx:codelistItem>
+   <!--=== DS_AssociationTypeCode ===-->
```
Comment (TylerHeald1) [inline umm-lib/resources/schema/iso_mends/resources/Codelist/eosCodelists.xml:190]: For clarification, what do the XML changes have to do with linter issues?

File: acl-lib/dev/user.clj [hunk 2]
```diff
@@ -1,12 +1,3 @@
-(ns user
```
Comment (TylerHeald1) [inline acl-lib/dev/user.clj:1]: Does removing these :require, :use, and reset affect local dev at all?

Comment (eereiter) [reply acl-lib/dev/user.clj:1]: Nothing.  Stuff from a previous merge got it.  I will need to redo this branch.

Comment (eereiter) [reply acl-lib/dev/user.clj:1]: No.  My guess is that someone who was developing the acl lib used things in the user namespace, then got rid of the code in the user namespace, but never got rid of the require statements.

File: umm-lib/resources/schema/iso_mends/resources/Codelist/eosCodelists.xml (hunk 1 above)
Comment (eereiter) [reply umm-lib/resources/schema/iso_mends/resources/Codelist/eosCodelists.xml:190]: Nothing.  It looks like the code from my previous issue that I was working got into this PR.  I will need to redo my changes and issue a new PR.

**Pull request closed without merging**